lib/matrix_sdk/api.ex

```diff
@@ -371,6 +371,15 @@ defmodule MatrixSDK.API do
     |> @http_client.do_request()
   end
 
+  @doc """
+  Redacts a room event with a reason.
+  """
```

Comment:
Thanks for documenting everything, could you add the `@spec` and an example as well?